client/src/components/Reservation.test.jsx

```diff
@@ -5,6 +5,6 @@ import Reservation from './Reservation';
 
 describe('Reservation Component', () => {
   test('should render correctly', () => {
-    expect(shallow(<Reservation />)).toMatchSnapshot();
+    expect(shallow(<Reservation id={300} />)).toMatchSnapshot();
```

Comment:
though this test is just testing for a snapshot, your actual render passes in the id 100 as the prop but here you pass it in 300, is there a reason why, just curious. is it because its just testing for the div?

Reply:
It was just a placeholder, but I have updated to 100 to match what is passed in by my index.jsx on instantiation